src/components/HomepageHeader.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+interface HomepageHeaderProps {
+  userName: string;
+  userImage: string;
+}
+
+export const HomepageHeader: React.FC<HomepageHeaderProps> = ({
+  userName,
+  userImage,
+}) => {
+  return (
+    <div className="flex w-screen h-10 bg-dark pl-5 pr-5 pt-8 text-white justify-between items-center">
```

Comment:
- doesn't need a height, the height is determined by the content
- The component has no padding